Packs/GitLab/Integrations/GitLabv2/GitLabv2.py

```diff
@@ -0,0 +1,1608 @@
+            'assignee': ['name', 'id'],
+            'created_at': None,
+            'updated_at': None,
+            'closed_at': None,
+            'state': None,
+            'severity': None
+        }
+
+    if object_name == 'Merge Request':
+        return{
+            'id': None,
+            'iid': None,
+            'title': None,
+            'description': None,
+            'state': None,
+            'author': ['name', 'id'],
+            'created_at': None,
+            'closed_at': None,
+            'source_branch': None,
+            'target_branch': None
+        }
+    if object_name == 'Commit':
+        return{
+            'id': None,
+            'short_id': None,
+            'title': None,
+            'message': None,
+            'author': ['name'],
+            'created_at': None
+        }
+
+    if object_name == 'Issue Note':
+        return{
+            'id': None,
+            'created_at': None,
+            'updated_at': None,
+            'body': None,
+            'noteable_iid': None,
+            'author': ['name', 'id']
+        }
+
+    if object_name == 'Merge Request Note':
+        return{
+            'id': None,
+            'created_at': None,
+            'updated_at': None,
+            'body': None,
+            'noteable_iid': None,
+            'author': ['name', 'id']
+        }
+
+    if object_name == 'Project':
+        return{
+            'id': None,
+            'description': None,
+            'name': None,
+            'created_at': None,
+            'default_branch': None,
+            'namespace': ['name', 'id']
+        }
+    return {}
+
+
+def partial_response(response: list, object_type: str):
+    '''
+    This function filters the raw response from the API according to the dict of fields given.
+    input: raw response which is a list of dictionaries, fields for the context data display.
+    output: partial dictionary results.
+    '''
+    partial_response: List[Dict[str, Any]] = []
+    fields = partial_response_fields(object_type)
+    for raw_dict in response:
+        partial_dict: Dict[str, Any] = {}
+        for field_key, field_dict_vals in fields.items():
+            if not(field_dict_vals):
+                partial_dict[field_key] = raw_dict.get(field_key, '')
+            elif raw_dict.get(field_key):
+                temp_dict_vals: Dict[str, Any] = {}
+                for val in field_dict_vals:
+                    temp_dict_vals[val] = raw_dict.get(field_key, {}).get(val, '')
+                partial_dict[field_key] = temp_dict_vals
+        partial_response.append(partial_dict)
+    return partial_response
+
+
+def verify_project_id(client: Client, project_id: Any) -> Boolean:
+    '''
+    This function verify that the user can access the project.
+    input: project_id
+    output: True is the project_id is valid, otherwise an error will occur.
+    '''
+    project_id = arg_to_number(project_id)
+    if not project_id or project_id < 0:
```

Comment:
ok. But since it can still be None and fails on lint because  I use this value, I added :
project_id = arg_to_number(project_id, required=True) or -1 

so it won't fail ok?